code/__defines/ai.dm

```diff
@@ -0,0 +1,22 @@
+// TODO: FSM/decl based stances
+#define STANCE_NONE                   0
```

Comment:
is there a reason these have to be numbers, aside from `STANCE_NONE` being falsey? might be a good idea to, between now and whenever these stances are replaced with decls, change them to strings to reduce collisions esp. with modpacks and such